app/models/credit.rb

```diff
@@ -38,4 +40,17 @@ def item_name
     # TODO: change it depending on invoice template
     credit_note.invoice.number
   end
+
+  def invoice_coupon_display_name
+    return nil if applied_coupon.blank?
+
+    coupon = applied_coupon.coupon
+    suffix = if coupon.percentage?
+               "#{sprintf("%.2f", applied_coupon.percentage_rate)}%"
```

Comment:
Layout/IndentationWidth: Use 2 (not 11) spaces for indentation. (https://rubystyle.guide#spaces-indentation)